dts/arm/microchip/mec1501hsz.dtsi

```diff
@@ -169,6 +169,36 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 		};
+		timer0: timer@40000c00 {
+			compatible = "microchip,xec-timer";
+			clock-frequency = <48000000>;
+			reg = <0x40000C00 0x20>;
+			interrupts = <136 0>;
+			label = "TIMER_0";
+			max-value = <0xFFFF>;
+			prescaler = <0>;
+			status = "disabled";
+		};
+		timer1: timer@40000c20 {
+			compatible = "microchip,xec-timer";
+			clock-frequency = <48000000>;
+			reg = <0x40000C20 0x20>;
+			interrupts = <137 0>;
+			label = "TIMER_1";
+			max-value = <0xFFFF>;
+			prescaler = <0>;
+			status = "disabled";
+		};
+		timer3: timer@40000ca0 {
```

Comment:
what about timer2?

Reply:
There are four timers and the third one is being used as the system clock. So it is not being declared here to avoid being used as a counter.